src/components/Header.tsx

```diff
@@ -0,0 +1,29 @@
+export const Header = () => {
+  return (
+    <nav className="nav flex justify-between items-center w-full bg-[#90E9EF] text-black p-4">
```

Comment:
You might consider using the method Nick used to save colors in the tailwind config file. I know this is already something you mentioned, I'm just putting it here to document it.